pyopencl/cl/pyopencl-complex.h

```diff
@@ -77,6 +77,13 @@
     ; \
   } \
   \
+  inline TP TPROOT##_fma(TP c, TP a, TP b)            \
+  { \
+    return TPROOT##_new( \
+        (c.real + a.real*b.real) - a.imag*b.imag, \
+        (c.imag + a.real*b.imag) + a.imag*b.real); \
+  } \
+  \
```

Comment:
Want to add a test, or at least something that exercises this functionality?

Reply:
Done in 6d5bbf4